timesketch/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2014 Google Inc. All rights reserved.
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
No need to change the year; the most earliest is the most relevant.
As discussed since the code is APL you could just entirely remove the APL header from the source files.